app/controllers/issues_controller.rb

```diff
@@ -0,0 +1,26 @@
+class IssuesController < ApplicationController
+  # Next line should be removed in next issue
```

Comment:
Delete this comment. It isn't have enough information. In case person won't delete this line "in the next issue" you will do code review and could ask to delete it